x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/builtin_action_types/cases_webhook.ts

```diff
@@ -386,13 +382,21 @@ export default function casesWebhookTest({ getService }: FtrProviderContext) {
             .expect(200);
 
           expect(proxyHaveBeenCalled).to.equal(true);
+          const { pushedDate, ...dataWithoutTime } = body.data;
+          body.data = dataWithoutTime;
+
+          // check timestamp
+          const timestampTime = new Date(pushedDate).getTime();
+          const timeNow = Date.now();
+          const timeMinuteAgo = timeNow - 1000 * 60;
+          expect(timestampTime).to.be.within(timeMinuteAgo, timeNow);
+
```

Comment:
This is subject to flakiness. It is difficult to predict time in tests. I think it should be removed.